backend/common/db/migrations/versions/2026_04_05_1200-add_password_reset_tokens.py

```
@@ -0,0 +1,107 @@
+
+    for index_name in (
+        "idx_password_reset_tokens_user_id",
+        "idx_password_reset_tokens_token",
+    ):
+        if dialect == "postgresql":
+            conn.execute(sa.text(f"DROP INDEX IF EXISTS {index_name}"))
+        else:
+            try:
+                op.drop_index(index_name, table_name="password_reset_tokens")
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)